helpers.js

```diff
@@ -14,8 +15,86 @@ function stripCommas(invalidNumber) {
   return parseFloat(invalidNumber.replace(',', '').trim()).toFixed(2);
 }
 
+/**
```

Comment:
I'm not a fan of helper classes in general, I find they become bloated with time. Keeping utility functions in here is fine but I think command-specific functions like `getEventsFor`, `getAstronomyPictureOfTheDay`, etc. are not utilitarian enough to be here. They should be moved to the respective files in which they're used.